deploy/iqe-trigger-integration.yml

```
@@ -177,3 +179,5 @@ parameters:
   value: '1920x1080'
 - name: IQE_PARALLEL_ENABLED
   value: "false"
+- name: SE_NODE_SESSION_TIMEOUT
+  value: 600
```

`error processing template: [None]: error: failed to read input object (not a Template?): unable to decode "STDIN": json: cannot unmarshal number into Go struct field Parameter.parameters.value of type string
`
```suggestion
  value: "600"
```